src/screens/Settings/calendar.module.scss

```diff
@@ -13,6 +13,34 @@
     .rbc-time-content {
       overscroll-behavior: none;
     }
+
+    .rbc-today {
+      background-color: #feeef2;
+      opacity: 0.7 !important;
+    }
+    .rbc-toolbar,
+    .rbc-btn-group {
+      & button {
```

Comment:
The `&` is unnecessary.

Reply:
Yes you are right. It was just what I found in the documentation. I will remove that